x-pack/plugins/code/public/components/file_tree/file_tree.tsx

```diff
@@ -93,12 +92,28 @@ export class CodeFileTree extends React.Component<Props> {
             onClick={onClick}
           >
             {forceOpen ? (
-              <FolderOpenTriangle onClick={onFolderClick} />
+              <EuiIcon
+                type="arrowDown"
+                size="s"
+                color="subdued"
+                style={{ marginRight: '.5rem' }}
+                onClick={onFolderClick}
+              />
             ) : (
-              <FolderClosedTriangle onClick={onFolderClick} />
+              <EuiIcon
+                type="arrowRight"
+                size="s"
+                color="subdued"
+                style={{ marginRight: '.5rem' }}
+                onClick={onFolderClick}
+              />
             )}
-            <EuiIcon type={forceOpen ? 'folderOpen' : 'folderClosed'} />
-            <DirectoryNode>{`${node.name}/`}</DirectoryNode>
+            <EuiIcon type={forceOpen ? 'folderOpen' : 'folderClosed'} color="subdued" />
+            <DirectoryNode>
+              <EuiText size="s" grow={false} style={{ display: 'inline-block' }}>
```

Comment:
Same here. There is an eui utility class for this. https://elastic.github.io/eui/#/utilities/css-utility-classes